testing/tests/test_consistency_checker.py

```diff
@@ -85,46 +85,40 @@ def test_workload_event_without_container():
         _Event("foo-pebble-custom-notice", container=Container("foo"), notice=notice),
         _CharmSpec(MyCharm, {"containers": {"foo": {}}}),
     )
-    check = CheckInfo("http-check")
-    assert_consistent(
-        State(containers={Container("foo", check_infos={check})}),
-        _Event("foo-pebble-check-failed", container=Container("foo"), check_info=check),
-        _CharmSpec(MyCharm, {"containers": {"foo": {}}}),
-    )
-    assert_inconsistent(
-        State(containers={Container("foo")}),
-        _Event("foo-pebble-check-failed", container=Container("foo"), check_info=check),
-        _CharmSpec(MyCharm, {"containers": {"foo": {}}}),
-    )
-    assert_consistent(
-        State(containers={Container("foo", check_infos={check})}),
-        _Event(
-            "foo-pebble-check-recovered", container=Container("foo"), check_info=check
-        ),
-        _CharmSpec(MyCharm, {"containers": {"foo": {}}}),
-    )
-    assert_inconsistent(
-        State(containers={Container("foo")}),
-        _Event(
-            "foo-pebble-check-recovered", container=Container("foo"), check_info=check
-        ),
-        _CharmSpec(MyCharm, {"containers": {"foo": {}}}),
-    )
-    # Ensure the check is in the correct container.
-    assert_inconsistent(
-        State(containers={Container("foo", check_infos={check}), Container("bar")}),
-        _Event(
-            "foo-pebble-check-recovered", container=Container("bar"), check_info=check
-        ),
-        _CharmSpec(MyCharm, {"containers": {"foo": {}, "bar": {}}}),
-    )
-    assert_inconsistent(
-        State(containers={Container("foo", check_infos={check}), Container("bar")}),
-        _Event(
-            "bar-pebble-check-recovered", container=Container("bar"), check_info=check
-        ),
-        _CharmSpec(MyCharm, {"containers": {"foo": {}, "bar": {}}}),
+
+
+def test_check_info_container_matches_event():
```

Comment:
These are the same as above but:
* They add they layer, which they probably should have been doing before, and now have to.
* They're better labelled, so it's easier to see what inconsistency is being checked with each one.
* They aren't in the "no container" test, which they shouldn't have been.
* They're done in a loop rather than repeating for recovered and failed events.